tests/Queue/QueueWorkerTest.php

```diff
@@ -77,11 +77,11 @@ public function test_exception_is_reported_if_connection_throws_exception_on_job
         $this->exceptionHandler->shouldHaveReceived('report')->with($e);
     }
 
-    public function test_worker_sleeps_when_queue_is_empty()
```

Comment:
What's being tested here is runNextJob() and judging from the current code, runNextJob() is only used in "once" mode. daemon() method no longer depends on it.

There should be an extra test for daemon() mode though.